shared/templates/csv/sshd_lineinfile.csv

```diff
@@ -4,6 +4,7 @@ sshd_allow_only_protocol2,Protocol,2,false
 sshd_disable_empty_passwords,PermitEmptyPasswords,no,true
 sshd_disable_gssapi_auth,GSSAPIAuthentication,no,true
 sshd_disable_kerb_auth,KerberosAuthentication,no,true
+sshd_disable_pubkey_auth,PubkeyAuthentication,no,false
```

Comment:
We removed the old templating system, so these csv files got deleted and are obsolete. Can you please rebase?

Reply:
will be rebased shortly.